src/shapes/itext.class.ts

```diff
@@ -406,6 +398,16 @@ export class IText extends ITextClickBehaviorMixin<ITextEvents> {
     return boundaries;
   }
 
+  /**
+   * Renders cursor on context Top, outside the animation cycle, on request
+   * Used for the drag/drop effect.
+   * If contextTop is not available, do nothing.
+   */
+  renderCursorAt(selectionStart: number) {
```

Comment:
moved to its siblings in scope

Reply:
the only change that did not require a comment 🤣🤣🤣🤣

Reply:
🤣